python-avd/pyavd/_eos_designs/structured_config/network_services/port_channel_interfaces.py

```diff
@@ -42,7 +45,119 @@ def port_channel_interfaces(self: AvdStructuredConfigNetworkServicesProtocol) ->
+                        shutdown=not l3_port_channel.enabled,
+                        eos_cli=l3_port_channel.raw_eos_cli,
+                        flow_tracker=self.shared_utils.new_get_flow_tracker(
+                            l3_port_channel.flow_tracking, output_type=EosCliConfigGen.EthernetInterfacesItem.FlowTracker
+                        ),
+                        vrf=vrf.name if vrf.name != "default" else None,
+                        peer_type="l3_port_channel",
+                        peer_interface=l3_port_channel.peer_port_channel if l3_port_channel.peer_port_channel else None,
+                    )
+
+                    if l3_port_channel.ipv4_acl_in:
+                        port_channel_interface._update(access_group_in=l3_port_channel.ipv4_acl_in)
+                    if l3_port_channel.ipv4_acl_out:
+                        port_channel_interface._update(access_group_out=l3_port_channel.ipv4_acl_out)
+
+                    if "." not in l3_port_channel.name:
+                        port_channel_interface.switchport.enabled = False
+
+                    if l3_port_channel.ospf.enabled and vrf.ospf.enabled:
+                        port_channel_interface._update(
+                            ospf_area=l3_port_channel.ospf.area,
+                            ospf_network_point_to_point=l3_port_channel.ospf.point_to_point,
+                            ospf_cost=l3_port_channel.ospf.cost,
+                        )
+                        ospf_authentication = l3_port_channel.ospf.authentication
+                        if ospf_authentication == "simple" and (ospf_simple_auth_key := l3_port_channel.ospf.simple_auth_key) is not None:
+                            port_channel_interface._update(ospf_authentication=ospf_authentication, ospf_authentication_key=ospf_simple_auth_key)
+                        elif ospf_authentication == "message-digest" and (ospf_message_digest_keys := l3_port_channel.ospf.message_digest_keys) is not None:
+                            for ospf_key in ospf_message_digest_keys:
+                                if not (ospf_key.id and ospf_key.key):
+                                    continue
+                                port_channel_interface.ospf_message_digest_keys.append_new(
+                                    id=ospf_key.id,
+                                    hash_algorithm=ospf_key.hash_algorithm,
+                                    key=ospf_key.key,
+                                )
+                            if port_channel_interface.ospf_message_digest_keys:
+                                port_channel_interface.ospf_authentication = ospf_authentication
+
+                    ip_address = None
+                    if l3_port_channel.ip_address:
+                        ip_address = l3_port_channel.ip_address
+                    if ip_address:
+                        port_channel_interface.ip_address = ip_address
```

Comment:
You can just include ip_address in the object initialization in line 82. If it is None / Unset, it will not be set in the output.